reflex/components/base/script.py

```diff
@@ -0,0 +1,80 @@
+"""Next.js script wrappers and inline script functionality.
+
+https://nextjs.org/docs/app/api-reference/components/script
+"""
+from typing import Optional, Set
+
+from reflex.components.component import Component
+from reflex.event import EventChain, EventHandler
+from reflex.vars import BaseVar, Var
+
+
+class Script(Component):
+    """Next.js script component.
+
+    Note that this component differs from reflex.components.base.document.Script
+    in that it is intended for use with custom and user-defined scripts.
+    """
+
+    library = "next/script"
+    tag = "Script"
+    is_default = True
+
+    # Required unless inline script is used
+    src: Optional[Var[str]] = None
```

Comment:
We don't need to use `Optional` or set the default value to `None`, as that is done automatically for all props.

Can just do:
```python
src: Var[str]
strategy: Var[str]
...
```

Reply:
Okay thanks, good to know.